profiles/SamirPaul1.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://raw.githubusercontent.com/SamirPaul1/cv/main/assets/img/SamirPaul1-Overview.png",
```

Comment:
I recommend more of an image like this.

![image](https://user-images.githubusercontent.com/51391473/147397131-a108815e-874b-455c-ba15-224c954b44ca.png)

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/147397131-a108815e-874b-455c-ba15-224c954b44ca.png",
```